packages/reporter/src/runnables/runnables-store.ts

```diff
@@ -79,6 +80,7 @@ export class RunnablesStore {
 
     this.hasTests = numTests > 0
     this.hasSingleTest = numTests === 1
+    this.totalRunnables = numTests
```

Comment:
```suggestion
    this.totalTests = numTests
```
// nit: runnable could be a suite

Reply:
Done in [5e431a0](https://github.com/cypress-io/cypress/pull/25265/commits/5e431a05532e2116e048ad3468ede9aa41ff6eda)!